CMakeLists.txt

```diff
@@ -54,25 +36,42 @@ else()
 	target_link_libraries(${PROJECT_NAME} setupapi)
 endif()
 
-## Uncomment for example
+## Add example project
 add_executable(serial_example examples/serial_example.cc)
 add_dependencies(serial_example ${PROJECT_NAME})
 target_link_libraries(serial_example ${PROJECT_NAME})
 
 ## Include headers
 include_directories(include)
 
+## Install
+set(INSTALL_LIB_DIR lib)
+set(INSTALL_INCLUDE_DIR include)
+set(INSTALL_CMAKE_DIR share/serial/cmake)
+
 ## Install executable
 install(TARGETS ${PROJECT_NAME}
-    ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-    LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+    DESTINATION ${INSTALL_LIB_DIR}
+    EXPORT ${PROJECT_NAME}-targets
```

Comment:
Disclaimer: I'm not a CMake njina... I've tried quite a few things before coming up with this solution

More here: https://cmake.org/Wiki/CMake/Tutorials/Packaging#Packaging_and_Exporting

Also, according to this thread: https://cmake.org/pipermail/cmake/2009-November/033170.html
> What are you putting in the Config?  Typically the <pkg>Config.cmake
> file should just include the file produced by install(EXPORT).